src/main/java/frc/robot/commands/Presets/HighPresetSequence.java

```diff
@@ -32,5 +34,6 @@ public HighPresetSequence(TelescopeSubsystem telescope, RotatorSubsystem rotator
             crocodile.setWristToPositionCommand(WristPosition.CONE_SCORE).withTimeout(2)
             
         );
+        deadlineWith(new RunCommand(() -> crocodile.setIntakeSpeed(MathUtil.clamp((0.25 + intake.getAsDouble()), -1, 1) * multiplier)));
```

Comment:
Whoa, this doesn't do anything. Pop quiz for why, and why it's not a compiler nor runtime error.

It also seems unrelated to this PR.